console/src/channel/services/ontology.tsx

```diff
@@ -218,6 +222,30 @@ export const useDeleteAlias = (): ((props: Ontology.TreeContextMenuProps) => voi
     },
   }).mutate;
 
+export const useOpenCalculated =
+  () =>
+  ({ selection: { resources }, store }: Ontology.TreeContextMenuProps) => {
+    if (resources.length !== 1) return;
+    const resource = resources[0];
+    const tabKey = `editCalculated-${resource.id.key}`;
+    const layout = {
+      ...createCalculatedLayout,
+      key: tabKey,
+      windowKey: MAIN_WINDOW,
+      name: `Edit ${resource.name}`,
+      location: "mosaic" as const,
+      tab: {
+        closable: true,
+        editable: false,
+      },
+      args: {
+        channelKey: Number(resource.id.key),
+        channelName: resource.name,
+        expression: resource.data?.expression || "",
```

Comment:
Although this technically works, it's better to use nullish coalescing in these scenarios